tmt/utils.py

```diff
@@ -964,6 +964,10 @@ def public_git_url(url):
     authentication. For now just cover the most common services.
     """
 
+    # Using internal url instance transformtion is not supported
+    if re.match(r'ssh://.*osci\.redhat\.com/.*', url):
```

Comment:
I do not like this approach :( hardcoding some specific URL here. Wouldn't be better to say we do not support git ssh backend at all? Or to support it?

Reply:
I also don't like it, but theorerically for distgit it is safe to change `git` or `ssh` -> https

but also some public instances could have different path when using ssh, git, https, and probably whoever using ssh or git probably could have trouble to git clone without credentials.